packages/react/src/components/OverflowMenuItem/OverflowMenuItem-test.js

```diff
@@ -7,57 +7,241 @@
 
 import React from 'react';
 import OverflowMenuItem from '../OverflowMenuItem';
-import { mount } from 'enzyme';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
 
-const prefix = 'cds';
-const render = (props) =>
-  mount(<OverflowMenuItem closeMenu={jest.fn()} {...props} />);
+describe('OverflowMenuItem - RTL', () => {
+  describe('renders as expected', () => {
+    it('should support a className on the button node', () => {
+      render(
+        <OverflowMenuItem
+          closeMenu={jest.fn()}
+          itemText="one"
+          className="test-item"
+        />
+      );
+
+      expect(screen.getByRole('menuitem')).toHaveClass('test-item');
+    });
+
+    it('should support a className on the outermost node', () => {
+      const { container } = render(
+        <OverflowMenuItem
+          closeMenu={jest.fn()}
+          itemText="one"
+          wrapperClassName="test-item"
+        />
+      );
+
+      expect(container.firstChild).toHaveClass('test-item');
+    });
+
+    it('should spread extra props on the button node', () => {
+      render(
+        <OverflowMenuItem
+          closeMenu={jest.fn()}
+          itemText="one"
+          data-testid="test"
+        />
+      );
+
+      expect(screen.getByRole('menuitem')).toHaveAttribute(
+        'data-testid',
+        'test'
+      );
+    });
+
+    it('should support a className on the anchor node', () => {
+      render(
+        <OverflowMenuItem
+          closeMenu={jest.fn()}
+          href="carbondesignsystem.com"
+          itemText="one"
+          className="test-item"
+        />
+      );
+
+      expect(screen.getByRole('menuitem')).toHaveClass('test-item');
+    });
+
+    it('should spread extra props on the anchor node', () => {
+      render(
+        <OverflowMenuItem
+          closeMenu={jest.fn()}
+          itemText="one"
+          href="carbondesignsystem.com"
+          data-testid="test"
+        />
+      );
+
+      expect(screen.getByRole('menuitem')).toHaveAttribute(
+        'data-testid',
+        'test'
+      );
+    });
+
+    it('should call closeMenu on click', () => {
+      const closeMenu = jest.fn();
+      render(<OverflowMenuItem closeMenu={closeMenu} itemText="one" />);
+
+      userEvent.click(screen.getByRole('menuitem'));
+
+      expect(closeMenu).toHaveBeenCalled();
+    });
+
+    it('should be disabled', () => {
+      const { container } = render(
+        <OverflowMenuItem closeMenu={jest.fn()} itemText="one" disabled />
+      );
+
+      expect(container.firstChild).toHaveClass(
+        'cds--overflow-menu-options__option--disabled'
+      );
```

Comment:
Minor expansion on the coverage here since it also disables the child of the `li`

```suggestion
      expect(container.firstChild).toHaveClass(
        'cds--overflow-menu-options__option--disabled'
      );
      expect(container.firstChild.firstChild).toHaveProperty('disabled', true); 
```

Reply:
Good suggestion! Added :)